js/songs/new.html

```diff
@@ -0,0 +1,11 @@
+<h2>Add Song</h2>
+
+<form>
+  <input placeholder="Song Name" data-ng-model="songNewViewModel.song.title"/>
+  <input placeholder="Album Art url" data-ng-model="songNewViewModel.song.album_art"/>
+  <input placeholder="Artist" data-ng-model="songNewViewModel.song.artist"/>
+  <input placeholder="Genre" data-ng-model="songNewViewModel.song.genre"/>
+  <input placeholder="Created At" data-ng-model="songNewViewModel.song.created_at"/>
+  <input placeholder="Updated At" data-ng-model="songNewViewModel.song.updated_at"/>
+  <button type="button" data-ng-click="songIndexViewModel.create()">Create</button>
```

Comment:
Ah, I believe the issue is that you're referencing your index view model here (`songIndexViewModel`), not the one for your new state (`songNewViewModel`).